### PR TITLE
What: Uploaded more logos, corrected table, etc.

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* We were missing logos in the design readme
* the logo for the main readme was not darkmode friendly #687

This change addresses the need by:

* Created new logos
* Updated the table
* added the new dark mode safe logo to the main readme.

### CODE REVIEW

**Comment by ultrasaurus** [comment]
fyi -- to disable the spell-check (where appropriate) you can add a comment above wherever the unique words are (if you don't want to add to the repo-wide config, which would likely be the case here):
```
<!-- cSpell:disable -->
```

**Comment by ultrasaurus** [comment]
also, prolly even better, would be to ignore specific words



though common acronyms like "APAC" should prolly go in the repo-wide config `ci/spelling-config.json`

**Comment by ultrasaurus** [comment]
oh, I just realized ALL of the errors are in the main README -- will try to work on separate PR to fix that!

**Comment by ultrasaurus** [comment]
updated branch, spell check passes -- yay!

**Comment by ultrasaurus** [comment]
dark mode fixed, here's what the logo in the README looks like this PR:
image

File: design/README.md [hunk 1]
```diff
@@ -29,23 +29,14 @@
...
-        <td><img src="logo/cloud-native-security-horizontal-black.svg" width="200"></td>
-        <td><img src="logo/cloud-native-security-stacked-black.svg" width="95"></td>
-        <td><img src="logo/cloud-native-security-icon-black.svg" width="95"></td>
-</tr>
 <tr>
         <th>white</th>
-        <td><img src="logo/cloud-native-security-horizontal-white.png" width="200"></td>
-        <td><img src="logo/cloud-native-security-stacked-white.png" width="95"></td>
-        <td><img src="logo/cloud-native-security-icon-white.png" width="95"></td>
+        <td><img src="logo/Cloud-Native-Security-Horizontal-White.png" width="200"></td>
```
Comment (ultrasaurus) [inline design/README.md:34]: Please leave filenames consistent where possible.  Also, our style guide preferences all lower case filenames.

**Comment by ultrasaurus** [comment]
White logos should be displayed on grey background.  This is what it looks like for me:
image

**Review by ultrasaurus — CHANGES_REQUESTED** [review]
This is a great step forward, just a few small fixes and it'll be ready to merge.  See prior comments.

**Comment by ultrasaurus** [comment]
on spellcheck, I didn't wait long enough, I guess... weird that it showed passing for a minute.  

I would suggest adding darkmodesafe to `/ci/spelling-config.json` since it shows up in a couple of files, but also ok IMHO if you want to use cspell:ignore locally in both files.

File: design/README.md (hunk 1 above)
Comment (TheFoxAtWork) [reply design/README.md:34]: fixing.

**Comment by TheFoxAtWork** [comment]
pretty-logos

**Comment by TheFoxAtWork** [comment]
@ultrasaurus every change you requested should be resolved.  even added a preview of the new change!

File: ci/spelling-config.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
         "HIPAA",
         "CODEOWNERS",
         "explainability",
-        "APAC"
+        "APAC",
+        "darkmodesafe"
```
Comment (lumjjb) [inline ci/spelling-config.json:21]: i think the better fix for this is for cpell to ignore that line instead.

```suggestion
        "APAC",
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # CNCF Security Technical Advisory Group
 
-![Cloud Native Security
-logo](design/logo/cloud-native-security-horizontal-color.svg)
+<img src="design/logo/cloud-native-security-horizontal-darkmodesafe.svg" alt="Cloud Native Security logo" width="400"/>
```
Comment (lumjjb) [inline README.md:3]:
```suggestion
<!-- cspell:disable-next-line -->
<img src="design/logo/cloud-native-security-horizontal-darkmodesafe.svg" alt="Cloud Native Security logo" width="400"/>
```

**Review by lumjjb — COMMENTED** [review]
Looks good, trying another cspell fix

**Comment by lumjjb** [comment]
@ultrasaurus `<!-- cspell:disable-next-line -->` seems to work

**Comment by TheFoxAtWork** [comment]
@ultrasaurus can you verify the changes were applied so this can be merged?

**Comment by lumjjb** [comment]
Linked this PR to cspell issue https://github.com/cncf/tag-security/issues/724 to address, if not should be good to merge... @ultrasaurus if you can take a quick look and we can merge this PR

**Comment by ultrasaurus** [comment]
filenames should be all lower case:
image

not sure if that's the reason or if there are additional issues that cause broken images links: 
image

**Comment by TheFoxAtWork** [comment]
@ultrasaurus looks like i've finally fixed it, it wasn't accepting the file rename which was why the images were broken.

File: design/README.md [hunk 4]
```diff
@@ -1,17 +1,18 @@
-## TAG Security Logos
+## Security TAG Logos
 
-*Note: GitHub Flavored Markdown used in the Readme doesn't support background colors. The white logos below are displayed on the light grey of tables.*
+*Note: GitHub Flavored Markdown used in the Readme doesn't support background colors. The white logos below are displayed on the medium grey (#474756) in its row.*
```
Comment (ultrasaurus) [inline design/README.md:3]: Actually, it's not picking up that style.  It seems to be using a lighter grey
image

when I look at using the browser inspector, I see...
image

It is using the value `#f6f8fa`
image

**Review by ultrasaurus — COMMENTED** [review]
so close... not quite sure what the issue is, but dropping in a comment with what I'm seeing

**Comment by ultrasaurus** [comment]
@TheFoxAtWork looks like css is not supported by github, removed as part of the sanitization process: https://stackoverflow.com/questions/51956361/custom-css-file-for-readme-md-in-a-github-repo

I wonder if we could use the grey color swatch image as a background?  if github supports markup to overlay images or have background image or something

**Comment by ultrasaurus** [comment]
.... OR was the note at the top of the file supposed to indicate that it looks incorrect in github, but would look correct rendered as HTML in a normal browser... if so, maybe it would be easier to just include explicit HTML, I dunno.  Confusing.

**Comment by TheFoxAtWork** [comment]
@ultrasaurus i think i'll just render separate images just for the readme display with links to the source file without a background.

**Comment by TheFoxAtWork** [comment]
Screen Shot 2021-08-05 at 10 05 36 AM
okay - i've modified the table for the white logos so that they 'appear' to be displayed in the readme on the correct background with a link to take the user to the correct file with a transparent background.

**Review by ultrasaurus — APPROVED** [review]
Sorry for the delay.  Looks fine.  I don't love that it'll be harder to maintain, but reasonable trade-off.  In retrospect, the initial version against light gray wasn't so bad, and I think that would be ok too.

One note -- didn't check dark-mode, where black prolly has same problem, but I don't think that needs to block this merging in.

**Pull request merged**